ci/conda/cmake.sh

```diff
@@ -1,5 +1,14 @@
+#!/bin/sh
+
+if [ "$RUNNER_OS" == "Windows" ]; then
```

Comment:
It is weird that we have to explicitly specify SWIG_EXECUTABLE. We don't need to do that for GDAL builds

Reply:
This could probably be removed. It was in the mapserver feedstock and I carried it along from there.

Reply:
> This could probably be removed

indeed: https://github.com/MapServer/MapServer/pull/7109